examples/search/search-request.json

```diff
@@ -0,0 +1,27 @@
+{

```

Comment:
Why is there only one search request? ODR protocol should cover all use cases of search? How will consumer search for
- Provider's catalog
- Service Calendar of a particular provider's service
- Where are the tags based searches?